examples/codeviewer/shared/src/commonMain/kotlin/org/jetbrains/codeviewer/ui/editor/EditorView.kt

```diff
@@ -172,6 +180,7 @@ private fun codeString(str: String) = buildAnnotatedString {
         addStyle(AppTheme.code.value, strFormatted, Regex("[0-9]+"))
         addStyle(AppTheme.code.annotation, strFormatted, Regex("^@[a-zA-Z_]*"))
         addStyle(AppTheme.code.comment, strFormatted, Regex("^\\s*//.*"))
+        append("\n")
```

Comment:
Should it be wrapped to some condition? Something like
```kt
append(if (strFormatted.isNotEmpty()) strFormatted else "\n")
```

 Also, I'd left a comment with a link to the issue.

Reply:
It's better to leave as it is to keep separation between lines in copied text. Otherwise copied text will be joined in one single line.
Comments has been added.